### DISCUSSION TITLE
Best Practice

### DISCUSSION THREAD

**Discussion by heheda1212138** [LavaMoat/LavaMoat#775] [General]
Is there any project use lavamoat to protect their supply chain security? Or like some best practice with react/koa?

**Comment by naugtur** [comment]
For a large-scale project, you can look at https://github.com/MetaMask/metamask-extension/

We've not documented best practices yet, so we're looking for feedback from people using lavamoat in their projects. 
Let's keep the conversation going.

What do you use to build your project? Can you share more details about the setup?

BTW. You should be able to introduce `@lavamoat/allow-scripts` right away, that's where you start.
